modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMFormReportDataUtil.java

```diff
@@ -255,6 +256,13 @@ private static JSONObject _getPropertyLabelsJSONObject(
 	}
 
 	private static String _getValue(Value value) {
+		Locale portalUserLanguage = LocaleThreadLocal.getThemeDisplayLocale();
```

Comment:
Hi @MirnaGama, usually the variable name ends with the type name. In this case, it can be either `Locale locale` or `Locale themeDisplayLocale`.